### ISSUE TITLE
Project structure

### ISSUE THREAD

**Issue by polrodoreda** [sabueso/ticketator#89]
I open this issue to discuss about the structure of the project, how we update it and when we start it.
Maybe start creating separated apps:

- Tickets
- States
- Queues
- Company
- Groups
- Inventory
- Priorities
- Rights
- Core
    - Search
    - Calendar
- Accounts

And then having only one folder for templates and static files at the root.

**Comment by sabueso** [comment]
Good point, IMHO, we need some operational structure first, to know exactly what part do what.
@pasqualguerrero what's your opinion?
